alvr/client_core/src/platform/android.rs

```diff
@@ -430,57 +455,20 @@ pub fn video_decoder_split(
             }
 
             while running.value() {
-                if image_queue.lock().len() > available_buffering_frames {
-                    warn!("Video frame queue overflow!");
-
-                    image_queue.lock().clear();
-
-                    continue;
-                }
-
-                let mut acquired_image_ref = acquired_image.lock();
-
                 match decoder.dequeue_output_buffer(Duration::from_millis(1)) {
                     MediaCodecResult::Ok(buffer) => {
                         // The buffer timestamp is actually nanoseconds
-                        let timestamp = Duration::from_nanos(buffer.presentation_time_us() as _);
+                        let presentation_time_us = buffer.presentation_time_us();
```

Comment:
This will break head pose matching. When rebuilding the Duration it will be missing nanosecond precision and the equality check will fail.

Reply:
As you can see in the callback function, I'm using the same timestamp. It's the same nanoseconds in both cases without loss in precision.

Reply:
I see. Please rename this to `presentation_time_ns` to make this clearer.